pkg/keystore/keystore.go

```diff
@@ -136,6 +136,52 @@ func (ks *Keystore) MaskTrustedKeySystemRoot(fingerprint string) (string, error)
 	return dst, ioutil.WriteFile(dst, []byte(""), 0644)
 }
 
+func (ks *Keystore) TrustedKeyPrefixExists(prefix string, r io.ReadSeeker) (bool, error) {
+	defer r.Seek(0, os.SEEK_SET)
+
+	pubkeyBytes, err := ioutil.ReadAll(r)
+	if err != nil {
+		return false, err
+	}
+	entityList, err := openpgp.ReadArmoredKeyRing(bytes.NewReader(pubkeyBytes))
+	if err != nil {
+		return false, err
+	}
+	pubKey := entityList[0].PrimaryKey
```

Comment:
I am not sure... the documentation says:
```
ReadKeyRing reads one or more public/private keys
```
But it might be possible to craft an empty key which would make rkt crash? I will make this more robust. This pattern existed in other functions as well.